addons/account/views/account_move_views.xml

```diff
@@ -719,14 +719,9 @@
                                 invisible="not to_check" data-hotkey="k" />
                         <field name="state" widget="statusbar" statusbar_visible="draft,posted"/>
                     </header>
-                    <div class="alert alert-warning mb-0" role="alert"
-                         invisible="state != 'draft' or not duplicated_ref_ids">
-                        Warning: this bill might be a duplicate of
-                        <button name="open_duplicated_ref_bill_view"
-                                type="object"
-                                string="one of those bills"
-                                class="btn btn-link p-0"
-                        />
+                    <div class="alert alert-warning mb-0 w-100" invisible="state != 'draft' or not duplicated_ref_ids"  role="alert">
+                        <span> Warning: this bill might be a duplicate of</span>
```

Comment:
- Is the first space in the span intended?
- Also when I looked in the UI, the first "button" wasn't spaced enough from the warning banner. ![image](https://github.com/odoo/odoo/assets/52834046/d31f41f8-9773-4af3-a4d1-353bbdb32c4b)
I think this might do the trick:
```suggestion
                        <span class="mr-1">Warning: this bill might be a duplicate of</span>
```

Reply:
fixed